PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -344,7 +351,7 @@ struct HfCorrelatorDplusHadrons {
         for (unsigned int iclass = 0; iclass < classMl->size(); iclass++) {
           outputMl[iclass] = candidate.mlProbDplusToPiKPi()[classMl->at(iclass)];
         }
-        entryDplusCandRecoInfo(hfHelper.invMassDplusToPiKPi(candidate), candidate.pt(), outputMl[0], outputMl[1]); // 0: BkgBDTScore, 1:PromptBDTScore
+        entryDplusCandRecoInfo(hfHelper.invMassDplusToPiKPi(candidate), candidate.pt(), outputMl[0], outputMl[idxBdtScore]); // 0: BkgBDTScore, 1:PromptBDTScore; 2:FDBDTScore
```

Comment:
On a second thought, this switching does not seem really right because the column that is filled is called `MlScorePrompt`. Why do you not store a vector with all the scores?

Reply:
Hi Vit, thank you. In this case, I need to modify it by adding a column for FDScore also to the below code.
https://github.com/AliceO2Group/O2Physics/blob/3ae0253c8e65b52b684ef180ee81317971705f76/PWGHF/HFC/DataModel/CorrelationTables.h#L375C19-L375C32
I need to modify the task correlation accordingly. I did it to avoid modification of central class (CorrelationTables.h). I will fix it.

Reply:
Adding a column should not be a problem.